sections/installation.js

```diff
@@ -187,6 +187,44 @@ export default [
   {
     type: "heading",
     size: "md",
+    content: "afterCreate",
+    href: "Installation-pooling-afterCreate"
+  },
+  {
+    type: "text",
+    content: "`afterCreate` callback (rawDriverConnection, done) is called when the pool aquires a new connection from the database server. done(err, connection) callback must be called for `knex` to be able to decide if the connection is ok or if it should be discarded right away from the pool."
+  },
+  {
+    type: "code",
+    language: "js",
+    content: `
+      var knex = require('knex')({
+        client: 'pg',
+        connection: {...},
+        pool: {
+          afterCreate: function (conn, done) {
+            // in this example we use pg driver's connection API
+            conn.query('SET timezone="UTC";', function (err) {
+              if (err) {
+                // first query failed, return error and don't try to make next query
+                done(err, conn);
+              } else {
+                // do the second query...
+                conn.query('SELECT set_limit(0.01);', function (err) {
```

Comment:
There is no need to make even a single query in `afterConnect`. Connection is already made when the callback is executed.

Only reason why in this example there are 2 queries is to show how multiple queries is done with that ugly and error prone callback syntax. As you can see those queries are just some setup queries how the connection which was just acquired should behave.